javatests/google/registry/flows/EppFlowInjectionModule.java

```diff
@@ -0,0 +1,46 @@
+package google.registry.flows;
+
+import org.joda.time.DateTime;
+
+import dagger.Module;
+import dagger.Provides;
+import google.registry.flows.EppException.UnimplementedOptionException;
+import google.registry.flows.domain.ExtraDomainValidation;
+import google.registry.model.domain.launch.LaunchCreateExtension;
+import google.registry.model.registry.label.ReservationType;
+
+@Module
+public class EppFlowInjectionModule {
```

Comment:
I agree, I initially had this in the testing directory and named something like "FakeEppFlowInjecitonModule" however dagger is not added to that package so it felt like an incorrect location. I can change to "TestingFlowIjnectionModule" but I think "EppTestFlowInjectionModule" would align better with the existing classes in the test flows directory. Thoughts?